tf-repo-mgmt/scripts/New-Repository.ps1

```diff
@@ -49,6 +49,12 @@ if (!$skipMetaDataCreation) {
   Set-Location -Path $tempPath
   git clone $governanceRepoUrl $tempRepoFolderName
   Set-Location -Path $tempRepoFolderName
+
+  $randomNumber = Get-Random -Minimum 1000 -Maximum 9999
+  gh fork --remote --fork-name "avm-terraform-governance-$randomNumber" --default-branch-only
+  gh repo set-default 'Azure/avm-terraform-governance'
+  git fetch upstream
+  git reset --hard upstream/main
```

Comment:
Perhaps if you forked and then cloned from the fork it wouldn't need this?

Reply:
This is in place because the fork may exist (there can only be one I think). We need to make sure we are up to date.

Reply:
I'll change to using this command - thanks. I'll still need the git reset bits though

Reply:
Ah, is the `git reset` syncing the main branch in the fork?